### PR TITLE
Add console.table

### PR DESCRIPTION
**Pull request by daynin**
- add console.table
- add tests

Node and Deno outputs:

### CODE REVIEW

File: js/cli_utils.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    out += `${" ".repeat(needed)}${cell}${" ".repeat(Math.ceil(needed))}`;
+    if (i !== row.length - 1) {
+      out += tableChars.middle;
+    }
+  }
+  out += tableChars.right;
+  return out;
+};
+
+export const cliTable = (head: string[], columns: string[][]) => {
```
Comment (ry) [inline js/cli_utils.ts]: Use `export function cliTable(...` 

https://github.com/denoland/deno_std#top-level-functions-should-not-use-arrow-syntax

Also add return type.

File: js/cli_utils.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  return str.replace(colorRegExp, "");
+}
+
+const countBytes = (str: string) => {
+  const normalized = removeColors(String(str)).normalize("NFC");
+
+  return encoder.encode(normalized).byteLength;
+};
+
+const renderRow = (row: string[], columnWidths: number[]) => {
```
Comment (ry) [inline js/cli_utils.ts]: https://github.com/denoland/deno_std#top-level-functions-should-not-use-arrow-syntax

Also add return type.

File: js/cli_utils.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  middle: " │ "
+};
+
+const colorRegExp = /\u001b\[\d\d?m/g;
+
+function removeColors(str: string) {
+  return str.replace(colorRegExp, "");
+}
+
+const countBytes = (str: string) => {
```
Comment (ry) [inline js/cli_utils.ts]: https://github.com/denoland/deno_std#top-level-functions-should-not-use-arrow-syntax

Also add return type.

File: js/cli_utils.ts [hunk 4]
```diff
@@ -0,0 +1,93 @@
+import { TextEncoder } from "./text_encoding";
```
Comment (ry) [inline js/cli_utils.ts]: Maybe call this file `js/console_table.ts`  ?

Comment (ry) [inline js/cli_utils.ts]: Please add copyright header
```
// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

**Comment by daynin** [comment]
@ry done

File: js/console_table.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  middle: " │ "
+};
+
+const colorRegExp = /\u001b\[\d\d?m/g;
+
+function removeColors(str: string): string {
+  return str.replace(colorRegExp, "");
+}
+
+const countBytes = (str: string): number => {
```
Comment (ry) [inline js/console_table.ts]: Use function syntax, not arrow for top-level functions.

File: js/console_table.ts [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  const normalized = removeColors(String(str)).normalize("NFC");
+
+  return encoder.encode(normalized).byteLength;
+};
+
+const renderRow = (row: string[], columnWidths: number[]): string => {
```
Comment (ry) [inline js/console_table.ts]: Use function syntax, not arrow for top-level functions.

**Comment by daynin** [comment]
@ry done

**Comment by daynin** [comment]
@ry thank you! :)

**Pull request merged**